src/Lecture2.hs

```diff
@@ -185,7 +274,9 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing [] = True
+isIncreasing [_] = True
+isIncreasing xs = and $ zipWith (\x y -> y > x) xs (tail xs)
```

Comment:
That's a pretty nice implementation using standard functions 👏🏻 

Alternatively, You can implement `isIncreasing` without using helper functions and by relying on laziness and smarter pattern-matching 🦥 

```suggestion
isIncreasing (x : y : ys) = x < y && isIncreasing (y : ys)
```